contracts/ResilientOracle.sol

```diff
@@ -67,7 +66,7 @@ contract ResilientOracle is PausableUpgradeable, AccessControlled, ResilientOrac
      * @notice Checks whether an address is null or not
      */
     modifier notNullAddress(address someone) {
-        require(someone != address(0), "can't be zero address");
+        if (someone == address(0)) revert("can't be zero address");
```

Comment:
If we're doing `revert`s anyway, would it make sense to revert with a custom error instead of a string? Custom errors are, afaik, more efficient and allow callers to catch errors selectively.